admin/tests/integration/components/complementary-certifications/attach-form/badges/list_test.js

```diff
@@ -12,14 +12,25 @@ module('Integration | Component | complementary-certifications/attach-badges/lis
 
   module('Without badges', function () {
     test('it should display an empty table of target profile badges', async function (assert) {
-      // given
-      // when
+      // given & when
       const screen = await render(hbs`<ComplementaryCertifications::AttachBadges::Badges::List />`);
 
       // then
       assert.dom(screen.getByRole('table', { name: 'Liste des résultats thématiques' })).exists();
       const rows = screen.getAllByRole('row');
       assert.strictEqual(rows.length, 1);
+    });
+  });
+
+  module('when the complementary certification has an external jury', function () {
+    test('it should display all columns', async function (assert) {
+      // given & when
+      const screen = await render(
+        hbs`<ComplementaryCertifications::AttachBadges::Badges::List @isComplementaryCertificationHasExternalJury={{true}} />`,
+      );
+
+      // then
+      const rows = screen.getAllByRole('row');
```

Comment:
Pour eviter tout les [0]
```suggestion
const [firstRow] = screen.getAllByRole('row');
````